packages/core/merge.js

```diff
@@ -27,7 +27,7 @@ function mergeOne(dst, src) {
     // is this a regular property?
     if (desc.hasOwnProperty('value')) { // eslint-disable-line
       // Do not merge properties with the 'undefined' value.
-      if (desc.value === undefined) return;
```

Comment:
Basically, hat this line is saying:

> This code
> ```js
> merge(anything, undefined)
> ```
> must do thing.

I believe the right fix would be to replace `return` with `continue`. Although, I'm not sure. I don't have time to check that. Sorry.

Reply:
Looking at this line:

https://github.com/stampit-org/stampit/blob/5b4cbbeb351e40edf3bd9c5e13d3c2d45c97581a/src/stampit.js#L99

I believe `continue` is the right way.